generated/nifake/session.py

```diff
@@ -560,9 +560,10 @@ def fetch_waveform_into(self, number_of_samples, waveform_data):
 
         Args:
             number_of_samples (int): Number of samples to return
+            waveform_data (numpy array of float64): Samples fetched from the device. Array should be numberOfSamples big.
 
         Returns:
-            waveform_data (list of float): Samples fetched from the device. Array should be numberOfSamples big.
+            waveform_data (numpy array of float64): Samples fetched from the device. Array should be numberOfSamples big.
```

Comment:
We don't really return anything, this is misleading.

Reply:
Should we also return it?